fuel-core/src/service/modules.rs

```diff
@@ -38,8 +40,17 @@ pub async fn start_modules(config: &Config, database: &Database) -> Result<Modul
     let block_producer = fuel_block_producer::Service::new(&config.block_producer, db).await?;
     let bft = fuel_core_bft::Service::new(&config.bft, db).await?;
     let sync = fuel_sync::Service::new(&config.sync).await?;
-    // let mut relayer = FuelRelayer::new(FuelRelayerConfig::default());
-    // let mut p2p = FuelP2P::new(FuelP2PConfig::default());
+
+    let mut relayer_builder = fuel_relayer::ServiceBuilder::new();
+    relayer_builder
+        .config(config.relayer.clone())
+        .db(Box::new(database.clone()) as Box<dyn RelayerDb>)
+        .import_block_event(block_importer.subscribe())
+        .private_key(
+            hex::decode("c6bd905dcac2a0b1c43f574ab6933df14d7ceee0194902bce523ed054e8e798b")
```

Comment:
Shouldn't this be part of the relayer config?

Reply:
I missed removing this. I have one last thing to do with the integration of `fuel-relayer` and that would be to include config in cli. This will be resolved there: https://github.com/FuelLabs/fuel-core/issues/447